packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -59,12 +59,24 @@ Use these steps to add or edit a template in one of the Okta-supported languages
 
 > **Note:** To access email customization with a free developer edition of an Okta org, you need to contact [Okta support](https://support.okta.com/help).
 
-1. In the Admin Console, go to **Customizations** > **Emails**.
-1. Click a default email template listed in the left pane.
-1. Click **Edit** to open the message in HTML. If you see **Add Translation** instead of **Edit**, the template is already customized. You need to make any additional edits in the default language version. Skip to step 2 in [Add translations](#add-translations).
-1. Select a language from the dropdown menu.
-1. Make your edits. See [Velocity Templating Language](#velocity-templating-language) for customization options.
-1. Click **Save**. The default language version of your edited message appears in the **Custom Email** table.
+1. In the Admin Console, go to **Customizations** > **Branding**.
+1. In the **Communications** section, in the **Emails** box, click **Edit**.
+1. In the **Email Templates** table, click a template name.
+   - The code editor is in preview mode.
+   - **Audience**: Select the email audience.
+      - All users
```

Comment:
should these items be bold, since they are selecting one? And do they need to be bulleted? You could just say:

Select the email audience: All users, Admin only, or Disabled for all.

Reply:
Good point. Thank you :)